deploy-agent/deployd/agent.py

```diff
@@ -416,7 +476,11 @@ def main():
         client = ServerlessClient(env_name=args.env_name, stage=args.stage, build=args.build,
                                   script_variables=args.script_variables)
 
+    uptime = utils.uptime()
     agent = DeployAgent(client=client, conf=config)
+    create_sc_increment('deployd.stats.ec2_uptime_sec',
+                        1.0,
```

Comment:
your not using uptime itself? and just have a value of 1?

Reply:
thank you! this was an oversight.